src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellFlyoutLayoutManager.cs

```diff
@@ -257,12 +256,11 @@ public void LayoutParallax()
 				{
 					topMargin = (float)UIApplication.SharedApplication.GetSafeAreaInsetsForWindow().Top;
 				}
+				else if (OperatingSystem.IsIOSVersionAtLeast(11))
```

Comment:
Why only on iOS 11 or higher?

Reply:
Good catch!! That was left over from when I was just checking safe area directly here, opposed to the header margin, which is only part of iOS11